integration/helper_endpoints_test.go

```diff
@@ -113,33 +114,33 @@ func authCallbackHandler(t *testing.T) func(rw http.ResponseWriter, req *http.Re
 	}
 }
 
-func tokenEndpointHandler(t *testing.T, oauth2 fosite.OAuth2Provider) func(rw http.ResponseWriter, req *http.Request) {
+func tokenEndpointHandler(t *testing.T, provider fosite.OAuth2Provider) func(rw http.ResponseWriter, req *http.Request) {
 	return func(rw http.ResponseWriter, req *http.Request) {
 		req.ParseForm()
 		ctx := fosite.NewContext()
 
-		accessRequest, err := oauth2.NewAccessRequest(ctx, req, &fosite.DefaultSession{})
+		accessRequest, err := provider.NewAccessRequest(ctx, req, &oauth2.JWTSession{})
```

Comment:
It has to be `JWTSession` for `TestIntrospectToken` to be able to test any of the JWT strategies.